sihl-user/src/web.ml

```diff
@@ -0,0 +1,20 @@
+let user_from_token find_user ?(key = "user_id") read_token req
+    : Sihl.Contract.User.t option Lwt.t
+  =
+  let open Lwt.Syntax in
+  match Sihl.Web.Request.bearer_token req with
+  | Some token ->
+    let* user_id = read_token token ~k:key in
+    (match user_id with
+    | None -> Lwt.return None
```

Comment:
General info, `Lwt` has many returns we could use, here for example [`Lwt.return_none`](https://ocsigen.org/lwt/4.1.0/api/Lwt#VALreturn_none), but also `Lwt.return_unit` for tests, etc. (@mabiede showed me)

Reply:
Never started using them because it doesn't add a huge benefit imo, we can start using them if you want, no strong feelings about it.

Reply:
I care more about consistency, so we can just keep using normal `Lwt.return` and avoid the refactorings :)